lib/hls/hls_parser.js

```diff
@@ -2411,6 +2411,20 @@ shaka.hls.HlsParser = class {
     // stream is lazy-loaded.
     const mimeType = this.guessMimeTypeBeforeLoading_(type, codecs) ||
         this.guessMimeTypeFallback_(type);
+    const roles = [];
+    if (characteristics) {
+      for (const characteristic of characteristics.split(',')) {
+        roles.push(characteristic);
+      }
+    }
+
+    /**
+     * For running chrome in hls
+     */
+    if (!roles.length &&
+      kind === shaka.util.ManifestParserUtils.TextStreamKind.SUBTITLE) {
```

Comment:
> /home/runner/work/shaka-player/shaka-player/lib/hls/hls_parser.js:2425:6: ERROR - [JSC_UNDEFINED_VARIABLE] variable kind is undeclared
>   2425|       kind === shaka.util.ManifestParserUtils.TextStreamKind.SUBTITLE) {

There is no such variable "kind".  You may be thinking of the "kind" field in the anonymous object below.  You should pull that out into a variable first, and reference the variable below in the anonymous object.

Reply:
Agreed, let me make a fix.

Reply:
I edited this and fixed it myself.